src/database/schema/UserSchema.ts

```diff
@@ -182,14 +182,14 @@ class UserSchema implements BaseDbSchema {
     }
   };
 
-  isUserExists = async (
+  static isUserExists = async (
     db: SQLiteDatabase,
     userId: string,
     channelId: string,
     transaction?: Transaction
   ): Promise<boolean> => {
     try {
-      const query = `SELECT * FROM ${UserSchema.getTableName()} WHERE user_id = ? AND channel_id = ? LIMIT 1`;
+      const query = `SELECT user_id FROM ${UserSchema.getTableName()} WHERE user_id = ? AND channel_id = ? LIMIT 1`;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from instance method to static method for `isUserExists` is a good one, as it allows the method to be called directly on the class without needing an instance of it. Also, updating the query to select only `user_id` instead of `*` is a good optimization, as it reduces the amount of data fetched from the database when we only need to check if a user exists.
```diff
-  const query = `SELECT * FROM ${UserSchema.getTableName()} WHERE user_id = ? AND channel_id = ? LIMIT 1`;
+  const query = `SELECT user_id FROM ${UserSchema.getTableName()} WHERE user_id = ? AND channel_id = ? LIMIT 1`;
```
However, please ensure that all calls to `isUserExists` have been updated to use the static method syntax.


<!-- This is an auto-generated comment by OSS CodeRabbit -->